macos/generate.sh

```diff
@@ -3,7 +3,7 @@ source ../common/arch.sh
 
 title_case()
 {
-    python -c 'import sys; print " ".join(map(str.title, sys.argv[1:]))' $*
+    python -c 'import sys;import re; print " ".join(map(str.capitalize, [sys.argv[1].replace("_", "")]))' $*
```

Comment:
Could this be python3-compatible? I don't know for how long macOS keeps python2 in the future.
```suggestion
    python -c 'import sys;import re; print(" ".join(map(str.capitalize, [sys.argv[1].replace("_", "")])))' $*
```